### ISSUE TITLE
Encrypt admin_identifier

### ISSUE THREAD

**Issue by sh-ura** [dycons/participant-portal#2]
The JWT received from the Participant IdP (Keycloak) following authentication should have the `admin_identifier` property encrypted with a secret key. This secret key will be known by trusted microservices, who will use it to decrypt the admin_identifier.

**Acceptance Criteria**
- The JWT returned to a user by the Participant IdP contains an encrypted `admin_identifier` property. The `admin_identifier` can be decrypted using some secret key.

**Ticket dependencies**
dycons/participant-portal#1 Must be completed first.

**Comment by yuriy-pavlov** [comment]
WON'T DO
Closing due to repo archival.
